js/data/locale/zoneinfo/America/Santiago.json

```diff
@@ -1,17 +1,17 @@
 {
+    "c": "CL",
     "e": {
         "m": 4,
         "r": "0>2",
         "t": "0:0"
     },
     "f": "-04/-03",
+    "n": "Pacific SA {c} Time",
     "o": "-4:0",
     "s": {
         "m": 9,
-        "r": "0>2",
+        "r": "0>9",
```

Comment:
what is this data change mean? I try to check the tzdata 2022c release notes, but I can't find them.
https://mm.icann.org/pipermail/tz-announce/2022-August/000072.html

Reply:
From the file "NEWS" which is in each distribution. Note that we skipped from 2022a directly to 2022c:

```
Release 2022b - 2022-08-10 15:38:32 -0700

  Briefly:
    Chile's DST is delayed by a week in September 2022.
    Iran no longer observes DST after 2022.
    Rename Europe/Kiev to Europe/Kyiv.
```

Reply:
The rule "0>9" means the Sunday (0) that is greater than the 9th day of the month, so that could be anywhere from the 10th to the 16th.

Reply:
thanks to let me know!